### PR TITLE
docs: add dev note about feature toggles

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** I wanted a note somewhere about where ORA-specific feature toggles live. I added a brief note.

JIRA: [https://openedx.atlassian.net/browse/AU-284](https://openedx.atlassian.net/browse/AU-284)

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- **n/a** Translations and JS/SASS compiled
- **n/a** Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by jansenk — APPROVED** [review]

**Review by leangseu-edx — APPROVED** [review]

**Review by pomegranited — APPROVED** [review]
:+1: better late than never?

- [x] I tested this by reading the rendered docs change, and verified that the added documentation is still correct.
- [x] I read through the code
- [x] Updates documentation
